src/ctia/bundle/routes.clj

```diff
@@ -25,40 +60,21 @@
                 :header-params [{Authorization :- (s/maybe s/Str) nil}]
                 :query [q BundleExportQuery]
                 :summary "Export a record with its local relationships"
-                :capabilities
-                #{:list-campaigns
-                  :read-actor
-                  :read-malware
-                  :read-attack-pattern
-                  :read-judgement
-                  :read-sighting
-                  :list-sightings
-                  :list-relationships
-                  :read-coa
-                  :read-indicator
-                  :list-judgements
-                  :list-tools
-                  :list-indicators
-                  :read-feedback
-                  :list-verdicts
-                  :list-feedbacks
-                  :list-malwares
-                  :list-data-tables
-                  :list-incidents
-                  :read-campaign
-                  :list-attack-patterns
-                  :read-relationship
-                  :list-actors
-                  :read-investigation
-                  :read-incident
-                  :list-coas
-                  :read-tool
-                  :list-investigations
-                  :read-data-table
-                  :read-weakness
-                  :list-weaknesses
-                  :read-vulnerability
-                  :list-vulnerabilities}
+                :capabilities export-capabilities
+                :auth-identity identity
+                :identity-map identity-map
+                (ok (export-bundle
+                     (:ids q)
+                     identity-map
+                     identity
+                     (select-keys q [:include_related_entities :related_to]))))
+
+           (POST "/export" []
+                :return NewBundle
+                :header-params [{Authorization :- (s/maybe s/Str) nil}]
+                :body [q BundleExportQuery]
```

Comment:
To show all available values in swagger UI, I would keep `:include_related_entities` and `:related_to` as query parameters.